spec/model_spec.js

```diff
@@ -3363,7 +3370,37 @@ describe('Model', function () {
             ]
           });
         }).toThrow();
-      })
+      });
+
+      describe('loading subclass associations', function() {
+        it('loads subclass hasOne associations', function() {
+          Composition.load({
+            id: 'composition1000',
+            things: [
+              {
+                id: 'randomThing2',
+                type: 'AnotherChild',
+                widget: { id: 'widget1' }
+              }
+            ]
+          })
```

Comment:
You're not doing any assertions in these specs. Might want to assert that the associated objects are of the correct class.

Reply:
bah, you're right...I had them setup because those calls were failing prior to this change.  will add assertions